### PR TITLE
refactor: more flexible lex bot configuration tree

### PR DESCRIPTION
**Pull request by robert-bo-davis**
Primary reviewer:

## Description

This refactors lex configuration json into reusable chunks and sets up the lex bot configuration to be more DRY in the future while remaining flexible.

### CODE REVIEW

File: twilio-iac/helplines/configs/lex/en_US/intents/pre_survey.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    "conclusion_statement": {
+      "content": "We'll transfer you now. Please hold for a counsellor.",
+      "content_type": "PlainText"
+    },
+    "rejection_statement": {
+      "content": "Sorry, I am not able to assist at this time.",
+      "content_type": "PlainText"
+    },
+    "slots": {
+      "callerType": {
```
Comment (murilovmachado) [inline twilio-iac/helplines/configs/lex/en_US/intents/pre_survey.json:31]: can we change `callerType` to be `aboutSelf`?
That's the name the prepopulation code is waiting.

Also, `callerType` with value `Yes | No` doesn't make much sense

Comment (robert-bo-davis) [reply twilio-iac/helplines/configs/lex/en_US/intents/pre_survey.json:31]: I'm just showing a suggested config pattern to @acedeywin in this PR. Could you comment on his PR (#1484) once this is merged? That is his PR even though I opened it. I was just fixing some git branching issues. Sorry for all the confusion. We *really* need to get all of this merged up soon.

Comment (acedeywin) [reply twilio-iac/helplines/configs/lex/en_US/intents/pre_survey.json:31]: @murilovmachado, @robert-bo-davis  - I will effect this change on my PR.

**Review by acedeywin — APPROVED** [review]
LGTM

**Pull request merged**